contents/blog/the-posthog-array-1-42-0 copy.md

```diff
@@ -0,0 +1,79 @@
+---
+date: 2023-01-16
+title: "Array 1.43.0: HIGHLIGHTS"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Want to know more about what we're up to? Check out [our roadmap](/roadmap) to see what we're working on and what new beta features are available! You can also subscribe to [our Hogmail newsletter](/newsletter)!
+
+## Update guide for self-hosted users
+
+Any notes about async migrations, etc. go here.
+
+## PostHog 1.43.0 release notes
+
+**Release highlights:**
+
+- [New: Item 1](#new-item-1)
+- [New: Item 2](#new-item-2)
+- [New: Item 3](#new-item-3)
+- [Improved: Item 4](#improved-item-4)
+- [Apps: Item 3](#apps-item-5)
+
+### New: Item 1
+
+@Karl to add a sentence or two here. 
+
+### New: Item 2
+
+@Karl to add a sentence or two here. 
+
+### New: Item 3
```

Comment:
```suggestion
### New: Feature Flag variant overrides

Ever create a multivariate feature flag, and wanted to show some specific people the control variant only? Or, maybe you found a new cohort you'd like to add to the test variant?

Now, this is possible! You can manually override variants to the one you want on the flag edit screen.

This is also great for testing, as it can let you deterministically choose a variant, and lets you test across client-side and server-side feature flags.

```